themes/digital.gov/layouts/partials/core/communities.html

```diff
@@ -0,0 +1,53 @@
+{{/* Digital.gov Communities Component
+  Renders a list of Digital.gov communities on both the home page and communities page
+
+  @param {string} community_title - Title of the community
+  @param {string} community_deck - Short description of the community page.
+  @param {string} community_page - Either "list" for the community page or "home" for the home page. Dictates the layout of the community cards.
+*/}}
+
+{{/* Gets all communities pages */}}
+{{ $dg_communities := (where .context.Site.Pages "Section" "communities").ByTitle }}
+
+{{/* Gather only the pages with a weight greater than or equal to 1 */}}
+{{ $dg_communities := (where $dg_communities ".Params.weight" "ge" 1) }}
+
+{{/* Gather only the pages with a dg-emphasis key */}}
+{{ $dg_communities := (where $dg_communities ".Params.dg_highlight" "eq" true) }}
+
+
+<div
+  class="dg-communities-container{{ cond (eq .community_page "home") "-home" "-list" }}"
+>
+  <header
+    class="page-head {{ cond (eq .community_page "home") "dg-communities-title" "" }}"
```

Comment:
I like the `<header>` semantically because it matches with the `<header>`, `<section>`, `<header>`, `<section>` pattern we are using on the `/communities` page. I removed the parent `div` to avoid any conflicts.